uninstall-hook.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# uninstall-hook.sh
```

Comment:
Perhaps it's preferable to only remove one repo at a time, but then we'd need to do a multiline match, unless we change the setup output